src/cli/commands/cache.js

```diff
@@ -64,3 +64,7 @@ export const {run, setFlags} = buildSubCommands('cache', {
     }
   },
 });
+
```

Comment:
Can you please move it to top. Like we have for other commands: https://github.com/yarnpkg/yarn/blob/master/src/cli/commands/licenses.js#L13